Rubberduck.RibbonDispatcher/Sample Ribbons/RibbonLibrary/src/ErrorUtilities.bas

```diff
@@ -0,0 +1,22 @@
+Attribute VB_Name = "ErrorUtilities"
+Option Explicit
+
+Public Sub ReraiseError(ByVal Err As ErrObject, _
```

Comment:
Please remove this module. `Err.Raise Err.Number` is all that's needed to rethrow/re-raise a run-time error. This code mangles the `Err.Source` of the rethrown error; re-raising an error twice or more up the call stack results in a completely broken `Err.Source` at the handling site, and the benefits of `DisplayError` are marginal, not to mention the inconsistent shadowing of the `Err` global object.

Reply:
If Rubberduck has some other means of generating a stack trace from a VBA error great - otherwise this is the only means I am aware of to get that without a large amount of coding. I can't even begin to describe how much work these have saved me over the years, by providing a stack trace from which to begin
 problem diagnosis.

Reply:
`Err` is a function that returns a reference to singleton `ErrorObject`. Defining a parameter with the name `Err` results in `VBA.Err` being shadowed, and perhaps more importantly, there's no need to pass in a ByVal `ErrorObject` (to either routine), as it is always accessible through `VBA.Err` anyway.

When using `VBA.Err`, because it is a function, it is always better to use a `With Err` block, so as to avoid repeated function calls like "Err.Number & " - " Err.Description".

And finally, there are some situations in which the ErrorObject is cleared while bubbling, and it makes sense to save the error details to a separate class, so that the error can be reconstituted. I'll post my class for this to CR.